slimline.zsh

```diff
@@ -74,6 +74,12 @@ prompt_slimline_cwd() {
   echo "%F{${cwd_color}}%3~%f "
 }
 
+prompt_slimline_virtualenv() {
+  local parens_color='008'
```

Comment:
Yep! 
I would like to keep the color definition aligned with the text-color-codes of the project, unfortunately the "gray" color is not available in zsh, so I think that I'll go with "white" as default.